common/app/layout/Front.scala

```diff
@@ -365,10 +367,15 @@ object Front extends implicits.Collections {
     )
   }
 
+  private def applyShowMoreExperiment(faciaContainer: FaciaContainer)(implicit request: RequestHeader): FaciaContainer = {
+    val showMoreEnabled = ActiveExperiments.inOrOut(ShowMoreExperiment, in = false, out = true)
```

Comment:
could have been a simple 😀 
```
val showMoreEnabled = !ActiveExperiments.isParticipating(ShowMoreExperiment)
```

Reply:
True. 
I'd still like to add `inOrOut` since inlining if statements for non-boolean experiments isn't so nice :P

Reply:
it is still `val myVal = if(ActiveExperiments.isParticipating(ShowMoreExperiment)) something else somethingElse`. Not really complex

Reply:
👍